bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -160,5 +160,8 @@ <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
 {% endblock %}
 
 {% block js %}
+  {% if switch('stub-attribution-macos') %}
```

Comment:
Nit: from a user perspective all the critical UI logic is done in `firefox_new_fx_scene1`. Is there any reason why this `stub-attribution-macos` bundle needs to come before that? Could it be moved to last in the queue?